app/controllers/foods_controller.rb

```diff
@@ -1,70 +1,64 @@
 class FoodsController < ApplicationController
-  before_action :set_food, only: %i[ show edit update destroy ]
-
-  # GET /foods or /foods.json
-  def index
-    @foods = Food.all
-  end
-
-  # GET /foods/1 or /foods/1.json
-  def show
-  end
-
-  # GET /foods/new
   def new
     @food = Food.new
-  end
+    if params.key?(:recipe)
```

Comment:
- [ ] There is no index page for the food list and button I'm seeing. Please add that.

Reply:
we are team consist of three 
![Screenshot from 2023-09-23 00-18-50](https://github.com/emarkees/Recipe_App/assets/118057274/92c928a8-2477-4db0-8cd0-4ccd36d0c112)
cause we have inventory so the food only belongs to inventory or recipe and It can't be food alone
please take a look at the requirements and read them carefully

Reply:
:red_circle: [REQUIRED] As per the project requirements :arrow_down_small: 
![requi](https://github.com/emarkees/Recipe_App/assets/26736582/d8d028c7-8944-44e2-b1ac-0338b57d46e9)
and as requested by the previous reviewer, your project should include a food list page as it is part of the common interfaces which does not depend on the team size as specified in the project requirement :arrow_up_small:  
Here is the food list interface

![](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/food_index.png?token=GHSAT0AAAAAACFCOGPKECYQJGTKX26WZZNCZIOCUBA)

I would like to clarify a point regarding the relationship between food, inventory, and recipes based on the Entity-Relationship Diagram (ERD). You  have said, "We have inventory so the food only belongs to inventory or recipe and It can't be food alone" which seems to be wrong if we look closely at the diagram :arrow_down: 

![](https://github.com/microverseinc/curriculum-rails/raw/main/recipe-app/images/recipe_erd.png)

The ERD shows that both inventory and recipe are related to food through a many-to-many relationship, as evidenced by the "inventory_food" and "recipe_food" relationships.

This implies that food is indeed a stand-alone entity and exists independently. It is not limited to being a part of either inventory or a recipe exclusively. The ERD represents the flexibility of the system, allowing food items to be present in both inventory and recipes simultaneously or independently.